_config.yml

```diff
@@ -34,6 +34,12 @@ url: "https://fossrit.github.io"
 social:
   irc: https://web.libera.chat/#rit-foss
 
+# --- calendar feed settings ---
+calendar_feed:
+  url: "https://fossrit.github.io/feeds/calendar.ics"
+  description: "Upcoming FOSS@RIT events in iCalendar format"
+  timezone: "America/New_York"
+  
```

Comment:
Are these settings associated with a particular jekyll plugin? They dont seem to be referred to in the `calendar.ics` file and i dont see any similarly named plugins in `Gemfile` or `_config.yaml` that would indicate what these parameters are doing.